annex/restore-test/.gitignore

```diff
@@ -0,0 +1,3 @@
+.vagrant/*
+from_backup
```

Comment:
think this should be `shared/from_backup`

Reply:
Fixed